html/janus.js

```diff
@@ -469,15 +466,11 @@ function Janus(gatewayCallbacks) {
 			Janus.log("Using REST API to contact Janus: " + server);
 		}
 	}
-	var iceServers = gatewayCallbacks.iceServers;
-	if(iceServers === undefined || iceServers === null)
-		iceServers = [{urls: "stun:stun.l.google.com:19302"}];
+	var iceServers = gatewayCallbacks.iceServers || [{urls: "stun:stun.l.google.com:19302"}];
 	var iceTransportPolicy = gatewayCallbacks.iceTransportPolicy;
 	var bundlePolicy = gatewayCallbacks.bundlePolicy;
 	// Whether IPv6 candidates should be gathered
-	var ipv6Support = gatewayCallbacks.ipv6;
-	if(ipv6Support === undefined || ipv6Support === null)
-		ipv6Support = false;
+	var ipv6Support = gatewayCallbacks.ipv6 || false;
```

Comment:
See above: IMO this should be an explicit check on `true`, and a `false` otherwise.